jupyterlab/commands.py

```diff
@@ -239,6 +239,12 @@ def disable_extension(extension, app_dir=None, logger=None):
     _toggle_extension(extension, True, app_dir, logger)
 
 
+def get_npm_name():
+    """Get the appropriate npm executable name.
+    """
+    return 'npm.cmd' if os.name == 'nt' else 'npm'
```

Comment:
NB: I just solved this same issue in JupyterHub where I passed the `shell=True` arg to `check_call` which also works on Windows.